spec/lib/generators/forem/dummy/templates/config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
I did some testing and it seems when we specify an absolute minimum config block like this:
```ruby
SimpleForm.setup do |config|
end
```
... then simple_form does not complain about missing config. Sounds good, should I make the changes?